lib/facility.rb

```diff
@@ -1,14 +1,65 @@
+require 'date'
+
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_accessor :name, :address, :phone, :services, :registered_vehicles, :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(facility_info)
+    @name = facility_info[:name]
+    @address = facility_info[:address]
+    @phone = facility_info[:phone]
     @services = []
+    @registered_vehicles = []
+    @collected_fees = 0
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
   end
+
+  def register_vehicle(vehicle)
+    return nil if !@services.include?("Vehicle Registration")
+    if @services.include?("Vehicle Registration")
```

Comment:
If `!@services.include?("Vehicle Registration")` is `true`, this method will return `nil`, never reaching line 21. Therefore, line 21 will never evaluate to false, so you could remove this outer 'if' block.